docs/3.api/2.composables/third-party-capital/_dir.yml

```diff
@@ -0,0 +1,3 @@
+title: 'Composables'
+titleTemplate: '%s · Third Party Capital'
+icon: i-ph-arrows-left-right-duotone
```

Comment:
@huang-julien - added a subfolder here - let me know if this is correct.

Most importantly - I don't know what icon to use... let me know if you have ideas 😄

Reply:
Not sure what icon would represent third parties 😅